src/ncp/spinel.h

```diff
@@ -383,7 +383,7 @@ enum
 
     SPINEL_CAP_EXPERIMENTAL__BEGIN      = 2000000,
     SPINEL_CAP_EXPERIMENTAL__END        = 2097152,
-};
+} spinel_capability_t;
```

Comment:
Good point. We can use `unsigned int` as input for `spinel_capability_to_cstr()`... Changed this in the new pushed commit.